app/src/gui/components/workspace/notebooks.tsx

```diff
@@ -194,17 +198,35 @@ export default function NoteBooks() {
           </Button>{' '}
           tab and click the activate button.
         </Typography>
-        {showCreateNewNotebookButton && (
+        <div
+          style={{display: 'flex', justifyContent: 'space-between', gap: '8px'}}
+        >
+          {showCreateNewNotebookButton ? (
+            <Button
+              variant="contained"
+              onClick={() => history(ROUTES.CREATE_NEW_SURVEY)}
+              sx={{mb: 3, mt: 3, backgroundColor: theme.palette.primary.main}}
+              startIcon={<AddCircleSharpIcon />}
+            >
+              Create New {NOTEBOOK_NAME}
+            </Button>
+          ) : (
+            <div />
+          )}
           <Button
             variant="contained"
-            color="primary"
-            onClick={() => history(ROUTES.CREATE_NEW_SURVEY)}
+            disabled={refresh}
             sx={{mb: 3, mt: 3, backgroundColor: theme.palette.primary.main}}
-            startIcon={<AddCircleSharpIcon />}
+            startIcon={<RefreshOutlined />}
+            onClick={async () => {
+              setRefresh(true);
+              await syncRemoteProjects();
+              setRefresh(false);
+            }}
           >
-            Create New {NOTEBOOK_NAME}
+            Sync {NOTEBOOK_NAME}s
```

Comment:
Sync is not the right term here - overloaded with data sync - refresh would be better.